.github/workflows/ci.yaml

```diff
@@ -0,0 +1,28 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby-version:
+        - '2.6'
+        - '2.7'
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Ruby
+      uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: ${{ matrix.ruby-version }}
+        bundler-cache: true
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: '1.14'
+    - name: Run tests
+      run:
+      - travis_wait curl -L https://github.com/operator-framework/operator-sdk/releases/download/v0.18.2/operator-sdk-v0.18.2-$(uname
```

Comment:
travis_wait is not here

Reply:
Also can we move this into a bin/before_install for consistency with other repos?  before_install should be a shell script that checks for CI and installs any system packages and whatnot (Example: https://github.com/ManageIQ/manageiq-providers-openstack/blob/master/bin/before_install).  before_install should be before the `Set up Ruby` step (and so should be a pure shell script (Example: https://github.com/ManageIQ/manageiq-providers-openstack/blob/master/.github/workflows/ci.yaml#L30-L31)

Something like

```sh

Reply:
👍🏻 Fixed

Reply:
Can you do the bin/before_install script as well?

Reply:
@bdunne ^^

Reply:
Done